generators/insight/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const fountain = require('fountain-generator');
+const Insight = require('insight');
+const _ = require('lodash');
+
+module.exports = fountain.Base.extend({
+  initializing() {
+    this.insight = new Insight({
+      trackingCode: 'UA-77742614-1',
+      pkg: {
+        name: 'fountainjs.io',
+        version: require('../../package.json').version
```

Comment:
@Swiip 
in terminal will show
`? May fountainjs.io anonymously report usage statistics to improve the tool over time? (Y/n)`

Are you agree ?